dtmi/terasic/fcc/de10_nano_threshold-1.json

```diff
@@ -0,0 +1,219 @@
+      },
+      "description" : {
+        "en" : "Max and Min values of MPU9250 Gyroscope X axis"
+      },
+      "name": "gx",
+      "schema": "dtmi:Terasic:FCC:DE10_Nano_Threshold:MinMax:float;1",
+      "writable": true
+    },
+    {
+      "@id": "dtmi:Terasic:FCC:DE10_Nano_Threshold:MPU9250:GY;1",
+      "@type": "Property",
+      "displayName": {
+        "en": "Gyroscope Y Axis"
+      },
+      "description" : {
+        "en" : "Max and Min values of MPU9250 Gyroscope Y axis"
+      },
+      "name": "gy",
+      "schema": "dtmi:Terasic:FCC:DE10_Nano_Threshold:MinMax:float;1",
+      "writable": true
+    },
+    {
+      "@id": "dtmi:Terasic:FCC:DE10_Nano_Threshold:MPU9250:GZ;1",
+      "@type": "Property",
+      "displayName": {
+        "en": "Gyroscope Z Axis"
+      },
+      "description" : {
+        "en" : "Max and Min values of MPU9250 Gyroscope Z axis"
+      },
+      "name": "gz",
+      "schema": "dtmi:Terasic:FCC:DE10_Nano_Threshold:MinMax:float;1",
+      "writable": true
+    },
+    {
+      "@id": "dtmi:Terasic:FCC:DE10_Nano_Threshold:MPU9250:MX;1",
+      "@type": "Property",
+      "displayName": {
+        "en": "Magnetometer X Axis"
+      },
+      "description" : {
+        "en" : "Max and Min values of MPU9250 Magnetometer X axis"
+      },
+      "name": "mx",
+      "schema": "dtmi:Terasic:FCC:DE10_Nano_Threshold:MinMax:float;1",
+      "writable": true
+    },
+    {
+      "@id": "dtmi:Terasic:FCC:DE10_Nano_Threshold:MPU9250:MY;1",
+      "@type": "Property",
+      "displayName": {
+        "en": "Magnetometer Y Axis"
+      },
+      "description" : {
+        "en" : "Max and Min values of MPU9250 Magnetometer Y axis"
+      },
+      "name": "my",
+      "schema": "dtmi:Terasic:FCC:DE10_Nano_Threshold:MinMax:float;1",
+      "writable": true
+    },
+    {
+      "@id": "dtmi:Terasic:FCC:DE10_Nano_Threshold:MPU9250:MZ;1",
+      "@type": "Property",
+      "displayName": {
+        "en": "Magnetometer Z Axis"
+      },
+      "description" : {
+        "en" : "Max and Min values of MPU9250 Magnetometer Z axis"
+      },
+      "name": "mz",
+      "schema": "dtmi:Terasic:FCC:DE10_Nano_Threshold:MinMax:float;1",
+      "writable": true
+    }
+  ],
+  "schemas": [
+    {
+      "@id": "dtmi:Terasic:FCC:DE10_Nano_Threshold:MinMax:float;1",
+      "@type": "Object",
+      "fields": [
+        {
+          "displayName": {
+            "en": "Minimum Value"
+          },
+          "description" : {
+            "en" : "Minimum sensor value in float to ?????"
+          },
+          "name": "min",
+          "schema": "float"
+        },
+        {
+          "displayName": {
+            "en": "Maximum Value"
+          },
+          "description" : {
+            "en" : "Maximum sensor value in float to ?????"
+          },
+          "name": "max",
+          "schema": "float"
+        }
+      ]
+    },
+    {
+      "@id": "dtmi:Terasic:FCC:DE10_Nano_Threshold:MinMax:int;1",
```

Comment:
Is this interface schema needed? Doesn't look like its currently being used.

Reply:
They are used by tmi:Terasic:FCC:DE10_Nano_Threshold:MPU9250:Channel0;1 & tmi:Terasic:FCC:DE10_Nano_Threshold:MPU9250:Channel1;1

Reply:
Channel0/Channel1 are removed from the model.  Checking with Terasic/Intel team.